templates/repo/diff/box.tmpl

```diff
@@ -3,7 +3,7 @@
 	<div class="diff-detail-box diff-box">
 		<div class="gt-df gt-ac gt-fw">
 			{{if $showFileTree}}
-				<button class="diff-toggle-file-tree-button gt-df gt-ac not-mobile" data-show-text="{{ctx.Locale.Tr "repo.diff.show_file_tree"}}" data-hide-text="{{ctx.Locale.Tr "repo.diff.hide_file_tree"}}">
+				<button class="diff-toggle-file-tree-button hide-992 btn interact-fg" data-show-text="{{ctx.Locale.Tr "repo.diff.show_file_tree"}}" data-hide-text="{{ctx.Locale.Tr "repo.diff.hide_file_tree"}}">
```

Comment:
Does `not-mobile` fit this case? If the `not-mobile` works well, I think it doesn't need to introduce a new "hide-xxx" system.

Reply:
not mobile is 200px smaller. but it feels ok to show the file tree even up to the not-mobile breakpoint. 
I'll change this.